config/routes.rb

```diff
@@ -91,7 +91,7 @@
                     constraints: Curator::Middleware::ArkOrIdConstraint.new do
                       member do
                         concerns :administratable, only: [:show, :update], as: 'file_set_administrative', metastreamable_type: 'Filestreams::FileSet', controller: '/curator/metastreams/administratives'
-                        concerns :workflowable, only: [:show, :update], as: 'file_set_workflow', metastreamable_type: 'Filestreams::FileSet', controller: '/curator/metastreams/workflows'
+                        concerns :workflowable, only: [:show], as: 'file_set_workflow', metastreamable_type: 'Filestreams::FileSet', controller: '/curator/metastreams/workflows'
```

Comment:
Would this change prevent us from allowing updates via the API, for example if an admin user wants to set the publishing state on an object via the admin interface?